definitions/divorce/json/CaseEventToFields/CaseEventToFields-deemed-and-dispensed-nonprod.json

```diff
@@ -36,7 +36,8 @@
     "PageLabel": "Make Service Decision",
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "No"
+    "ShowSummaryChangeOption": "Yes",
+    "ShowSummaryContentOption": "Yes"
```

Comment:
Enabling CYA for CaseField

Reply:
"ShowSummaryContentOption": "Yes" definition would be removed in next commit, seems not to like 'Yes' or 'Y' values, CYA screen still works without it anyways. It would not be needed.